conf/modules/video_usb_logger.xml

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="computer_vision">
+  <doc>
+    <description>Logs attitude and position to a csv and images to jpeg files (only for linux)</description>
```

Comment:
Please remember that first line of the description (until dot) is used as name/title for the module in the generated dox.